ScorchApi/ScorchApiV2/Controllers/CharacterController.cs

```diff
@@ -161,8 +161,8 @@ public CharacterController(IOptions<AppSettings> appSettings)
             await _characterTable.DeleteItemAsync(characterId);
```

Comment:
This is badly merged. TJ added Sell item and delete item as two different endpoints